### PR TITLE
functionality for avg_ratings module

### PR DESCRIPTION
**Pull request by hyukkim0304**
@allisonc07 ready for review!

### CODE REVIEW

File: spec/server.test.js [hunk 1]
```diff
@@ -2,18 +2,18 @@ const request = require('supertest');
...
+    return request(server).get('/rooms/4/reviews').then(response => {
+      expect(response.statusCode).toBe(201);
+    });
+  });
+  test('Data should exist', () => {
+    return request(server).get('/rooms/1/reviews').then(response => {
+      expect(response).toBeDefined();
+    });
+  });
 });
```
Comment (allisonc07) [inline spec/server.test.js:19]: maybe add a test where you are seeing if the data returned matches the structure/format that you are expecting?

File: client/src/index.jsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ class App extends React.Component {
     return (
```
Comment (allisonc07) [inline client/src/index.jsx:30]: it won't let me add a comment on line 24, but you might want to add a .catch block for errors!

File: client/src/components/avg_rating/avg_rating.jsx [hunk 3]
```diff
@@ -1,7 +1,9 @@
 const React = require('react');
 
 function AvgRating(props) {
-  return <div>AVERAGE RATING</div>;
+  return <div>{props.data.length !== 0 && 
+                Math.round((props.data.map((data) => data.average).reduce((acc, cur) => acc + cur)) / props.data.length)}
```
Comment (allisonc07) [inline client/src/components/avg_rating/avg_rating.jsx:5]: I realize this is probably work in progress, but just curious, are you planning on rendering this as stars or a number?

**Review by allisonc07 — COMMENTED** [review]
Some minor questions! Looks good overall!

File: spec/server.test.js (hunk 1 above)
Comment (hyukkim0304) [reply spec/server.test.js:19]: will be added! I'm working on tests on different branch right now! Thanks!

Comment (allisonc07) [reply spec/server.test.js:19]: cool! sounds good 👍

**Comment by allisonc07** [comment]
yay! nice .catch 😎

**Pull request merged**